cli/command/stack/cmd.go

```diff
@@ -37,6 +41,10 @@ func NewTopLevelDeployCommand(dockerCli command.Cli) *cobra.Command {
 	cmd := newDeployCommand(dockerCli)
 	// Remove the aliases at the top level
 	cmd.Aliases = []string{}
-	cmd.Annotations = map[string]string{"experimental": "", "version": "1.25"}
+	cmd.Annotations = map[string]string{
+		"experimental": "",
+		"swarm":        "",
```

Comment:
I wasn't sure if I should add "kubernetes" for the top-level `docker deploy`; it's still marked "experimental" (and I think we should consider removing it)

Reply:
Yes, I think we should remove the top level one (and thus we don't need to annotate it)